doc/auto_techsupport_and_coredump_mgmt.md

```diff
@@ -36,13 +37,20 @@ Currently, techsupport is run by invoking `show techsupport` either by orchestra
 
 However if the techsupport invocation can be made event-driven based on core dump generation, that would definitely improve the debuggability. That is the overall idea behind this HLD. All the high-level requirements are summarized in the next section
 
+Another use case is to gather more information about the system in case there is a memory usage threshold crossed.
+SONiC dump generated after system reboots due to out of memory is not enough for debugging the issue
+as all the information about processes and their mem usage, smaps (/proc/PID/smaps) is lost.

```

Comment:
>lost [](http://example.com/codeflow?start=87&length=4)

Instead of invoking a full `show techsupport`, can we just save volatile information before reboot? For example, it makes no sense to save syslog during the memory pressure time. We could collect them after reboot.

Reply:
@qiluo-msft The user can configure the "since" parameter in CFG_DB and achive that result. If "since" is set to "0 sec" that would mean that no logs, sai, swss recs are recorded.